### PR TITLE
CC-Tools lab

### PR DESCRIPTION
**Pull request by samuelvenzi**
The CC-Tools package provides a relational-like framework for programming Hyperledger Fabric chaincodes.

### CODE REVIEW

**Comment by samuelvenzi** [comment]
About the sign-off DCO, we have done the following like instructed by @ryjones. Please let me know if that's ok.

https://github.com/GoLedgerDev/cc-tools-demo/pull/48/commits/d56a270f6226e2365f54a6e0e6cee82bb69b7824

https://github.com/GoLedgerDev/cc-tools/pull/27/commits/83dbb93628789f94e7084d4cc0c2768132200c2f

https://github.com/GoLedgerDev/cc-tools-doc/pull/5/commits/4b1a8f463a560149d14e84e2240edde78495c58c

**Comment by samuelvenzi** [comment]
@denyeart can you please confirm sponsorship?

**Comment by denyeart** [comment]
This looks like a great lab, I am happy to sponsor it.

Typically a lab will have a single repository, should we merge the 3 existing repositories into a single lab repository? @hyperledger-labs/lab-stewards

**Comment by tkuhrt** [comment]
> Typically a lab will have a single repository, should we merge the 3 existing repositories into a single lab repository? @hyperledger-labs/lab-stewards

We do have a number of labs with more than one repo. This would not be outside the norm if we wanted to keep all three repos. As long as they have the same prefix so that people know they go together that will work.

File: labs/CC-Tools.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# Short Description
+The CC-Tools package provides a relational-like framework for programming Hyperledger Fabric chaincodes.
+
+# Scope of Lab
+The CC-Tools project aims to provide an easy to use framework to develop Hyperledger Fabric chaincodes in Golang.
+
+CC-Tools provides a relational-like definitions of asset types, datatype validation, also allowing the creation of custom types, Fabric events management and easy transaction definition, with built-in arguments validation and callers restriction. Some pre-defined transactions includes standard method to create, delete, update and read assets.
+
+The CC-Tools-Demo repository provides a template for CC-Tools, featuring configurations and examples of asset types, transactions and events usage. It also features scripts to easily start or upgrade a network and a rest server API written using GIN, which provides access to endpoints to communicate with the chaincode (with routes using `fabric-sdk-go` and `fabric-gateway`) and automatic subscrition to CC-Tools events.
```
Comment (tkuhrt) [inline labs/CC-Tools.md:17]:
```suggestion
The CC-Tools-Demo repository provides a template for CC-Tools, featuring configurations and examples of asset types, transactions and events usage. It also features scripts to easily start or upgrade a network and a rest server API written using GIN, which provides access to endpoints to communicate with the chaincode (with routes using `fabric-sdk-go` and `fabric-gateway`) and automatic subscription to CC-Tools events.
```

File: labs/CC-Tools.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- https://github.com/andremacedopv
+- https://github.com/mikaellafs
+- https://github.com/jamillepp
+
+# Sponsor
+- https://github.com/denyeart - Hyperledger Fabric Maintainer and Release Manager, TOC Member
+
+# Pre-existing repository
+- https://github.com/GoLedgerDev/cc-tools
+- https://github.com/GoLedgerDev/cc-tools-demo
```
Comment (tkuhrt) [inline labs/CC-Tools.md:36]: I do not see a LICENSE file for this repo. What license is this repo? Can you add a LICENSE file?

File: labs/CC-Tools.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- https://github.com/GoLedgerDev/cc-tools-doc
```
Comment (tkuhrt) [inline labs/CC-Tools.md:37]: I do not see a LICENSE file for this repo. What license is this repo? Can you add a LICENSE file?

**Review by tkuhrt — CHANGES_REQUESTED** [review]

File: labs/CC-Tools.md (hunk 2 above)
Comment (samuelvenzi) [reply labs/CC-Tools.md:36]: @tkuhrt we just added the license file for cc-tools-demo

File: labs/CC-Tools.md (hunk 3 above)
Comment (samuelvenzi) [reply labs/CC-Tools.md:37]: @tkuhrt we just added the license file for cc-tools-doc

**Review by tkuhrt — APPROVED** [review]

**Review by nidhi-singh02 — APPROVED** [review]
Great lab! Thanks for submitting, will help developers in building CC.

**Comment by samuelvenzi** [comment]
@tkuhrt how would we proceed for repo transfer?

**Comment by tkuhrt** [comment]
Can you open an repo transfer issue here: https://github.com/hyperledger-labs/governance/issues/new/choose @samuelvenzi

**Pull request merged**